src/radiation/monte_carlo.cpp

```diff
@@ -280,13 +280,19 @@ TaskStatus MonteCarloSourceParticles(MeshBlock *pmb, MeshBlockData<Real> *rc,
             // Sample energy and set weight
             Real nu;
             int counter = 0;
+            Real prob;
```

Comment:
I wish I could do `Real prob = ` inside the `while {}` loop.

Reply:
Big Mood